### ISSUE TITLE
Consider re-instituting the STCP port (a la 8088)

### ISSUE THREAD

**Issue by guitarpicva** [FreeTAKTeam/FreeTakServer#194]
The simple connect to the STCP port which streams most TAK data like in the original server is a very valuable commodity for those of us who use the data to do "different" things, such as back-haul over disparate radio mediums.

Currently, it appears that to use the FTS in this way requires some level of "client connection".

If there is another way, I am all ears!  Federation?

**Comment by naman108** [comment]
thank you for your description based on the requirement for bi-direction communication either federation or a simulated client appears to be the best option however we can certainly look into implementing such a streaming functionality in future.

**Comment by guitarpicva** [comment]
There may be other simple data gatherers out there who find that feature very useful.  

Simulating a client isn't so hard; agreed.

I am trying to wade into Federation, but the requirement for TLS is a real turn off.  It's all on the same computer (in my case).  It's not like I haven't done TLS before.  It just slows my testing down, because of the PITA of the certs and keys.  Is there ANY documentation whatsoever on Federated connectivity in FTS?  The only thing I have run across is the screen shot stuff, which assumes you already know how to use it.  I have no idea what to expect.  I suppose it must have configurations for what portions of the TAK server are shared, etc.  Sort of like bridging MQTT brokers.  If anyone has any advice, do chime in please.

**Comment by brothercorvo** [comment]
is this part of 2.0 @naman108 ?

**Comment by naman108** [comment]
I believe this is covered under the repeater component

**Comment by brothercorvo** [comment]
> I believe this is covered under the repeater component

after further investigation it seems that this is a different feature.

> hat is Stream Control Transmission Protocol (SCTP)?
Stream Control Transmission Protocol (SCTP) is a connection-oriented [network protocol](https://www.techtarget.com/searchnetworking/definition/protocol) for transmitting multiple streams of [data](https://www.techtarget.com/searchdatamanagement/definition/data) simultaneously between two endpoints that have established a connection in a computer network.

> Sometimes known as next-generation [Transmission Control Protocol](https://www.techtarget.com/searchnetworking/definition/TCP) or TCPng, SCTP makes it easy to support a telephone connection over the internet.

> SCTP is an Internet Protocol (IP) [transport layer](https://www.techtarget.com/searchnetworking/definition/Transport-layer) protocol. It ensures reliable and in-sequence data transmission so that data units sent over the network arrive completely, and in the right sequence, to the application or user at the endpoint. For some transmissions, such as a file or record, this sequence preservation is essential.

so that would requires a new Service in FTS

**Comment by brothercorvo** [comment]
this framework could help https://github.com/P1sec/pysctp

**Comment by brothercorvo** [comment]
@naman108 this could be a good exercise for the new service support in DigitalPy

**Comment by naman108** [comment]
I agree, seems like a great chance to get a blank slate for service design, and could become our new default for tcp if all goes well.
